hypertrace-core-graphql-service/src/main/java/org/hypertrace/core/graphql/service/DefaultGraphQlServiceConfig.java

```diff
@@ -98,6 +101,11 @@ public int getGatewayServicePort() {
     return this.gatewayServicePort;
   }
 
+  @Override
+  public int getGatewayServiceRPCClientDeadline() {
```

Comment:
This doesn't make clear the expected units. Instead of changing the name, I'd recommend changing the type and returning a duration. This change can go all the way down, since the config lib natively supports durations.

That is:
```java
gatewayServiceClientDeadline = untypedConfig.getDuration(GATEWAY_SERVICE_RPC_CLIENT_DEADLINE);
```

Also, please remove rpc from the constant and getter names, since that's an impl detail and inconsistent with the config's actual key.

Reply:
Yes, that makes sense. RPC can be replaced with something else tomorrow and this will then diverge.

Reply:
Addressed. Used Duration as seconds and converted to millis during usage.